common/src/main/java/net/xolt/freecam/config/gui/DoubleSliderEntry.java

```diff
@@ -0,0 +1,150 @@
+package net.xolt.freecam.config.gui;
+
+import com.google.common.util.concurrent.AtomicDouble;
+import com.mojang.blaze3d.platform.Window;
+import me.shedaniel.clothconfig2.gui.entries.TooltipListEntry;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.client.gui.components.AbstractSliderButton;
+import net.minecraft.client.gui.components.AbstractWidget;
+import net.minecraft.client.gui.components.Button;
+import net.minecraft.client.gui.components.events.GuiEventListener;
+import net.minecraft.client.gui.narration.NarratableEntry;
+import net.minecraft.network.chat.Component;
+import net.minecraft.util.Mth;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+import static net.xolt.freecam.Freecam.MC;
+
+class DoubleSliderEntry extends TooltipListEntry<Double> {
```

Comment:
```suggestion
/**
* @see IntSliderEntry
*/
class DoubleSliderEntry extends TooltipListEntry<Double> {
```

Should probably reference the "original" source for both attribution and future reference.